From: Didier Raboud <odyx@debian.org>
Date: Tue, 9 Aug 2016 18:33:56 +0200
Subject: Tests: ignore errors triggered on ipv6-deprived hosts

Ignore the 'Address family not supported by protocol' error that breaks the
errorlines counting on buildds without a working IPv6 stack.
Origin: vendor
Patch-Name: tests-ignore-ipv6-address-family-not-supported.patch
---
 test/run-stp-tests.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/run-stp-tests.sh b/test/run-stp-tests.sh
index f5a98dd..f3a4321 100755
--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -1030,6 +1030,7 @@ count=`$GREP '^E ' $BASE/log/error_log | $GREP -v 'Unknown default SystemGroup'
        $GREP -v '(dnssd) stopped with status 1' | \
        $GREP -v 'loadFile failed: temp file: not a PDF file' | \
        $GREP -v 'Failed to connect to system bus' | \
+       $GREP -v -E 'Unable to open listen socket for address .* Address family not supported by protocol.' | \
        wc -l | awk '{print $1}'`
 if test $count != 33; then
 	echo "FAIL: $count error messages, expected 33."
